src/sparseml/transformers/export.py

```diff
@@ -97,7 +97,14 @@
     "tokenizer_config.json",
     "config.json",
 ]
-OPTIONAL_DEPLOYMENT_FILES: List[str] = [EXTERNAL_ONNX_DATA_NAME, "tokenizer.json"]
+
+OPTIONAL_DEPLOYMENT_FILES: List[str] = [
+    EXTERNAL_ONNX_DATA_NAME,
+    "tokenizer.json",
+    "special_tokens_map.json",
```

Comment:
to make this self-explains, I would put these 3 new files under a separate list with hint that it's for OPT model, before you do the merge.

Reply:
Crossed my mind, makes sense.

Reply:
agree